packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1049,6 +1049,26 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | TeamsCa
     this.handlers.onStopAllSpotlight();
   }
 
+  /* @conditional-compile-remove(breakout-rooms) */
+  public async returnToMainMeeting(): Promise<void> {
```

Comment:
I wonder if there is a way to attach this handler to a piece of state for the meeting? One thing I am thinking about is if we are to expose a handler like `switchCall` to allow contoso to have the adapter move to a call they are tracking a reference of. We would be maintaining two handlers that cause a switch call effect.

Reply:
That's a good point! I did originally have it something like switchCall but I chose to be more specific to breakout rooms. Being more general makes sense for a more usable function outside of breakout rooms and to avoid redundancy.
Do you prefer to be more general function here too @JamesBurnside?

Reply:
I think its best not to make this more general since that would then require contoso to maintain state of their own on the call, is there a way we can make this part of the breakoutRoomState feature but expose on the API to be like `adapter.breakoutRoomState.returnToMainMeeting();` just so we will keep track of the "main meeting" for them?

Thoughts @JamesBurnside?

Reply:
Really good discussion, really good thinking on having something generic. I think in this case its fine to keep it breakout room specific as you are right that we want to make the adapter as user friendly as possible. In the future I see switchCall using much the same code under the hood though to switch call vs "return" to  a call

Reply:
Sounds good! Will keep it specific for breakout rooms.